packages/fluent/scss/appbar/_variables.scss

```
@@ -0,0 +1,54 @@
+@use "../core/color-system" as *;
+@use "../core/variables" as *;
+
+/// Background color of the appbar.
+/// @group appbar
+$kendo-appbar-bg: var(--kendo-component-bg, initial) !default;
+/// Text color of the appbar.
+/// @group appbar
+$kendo-appbar-text: var(--kendo-component-text, initial) !default;
+/// Border color of the appbar.
+/// @group appbar
+$kendo-appbar-border: var(--kendo-component-border, initial) !default;
+
+/// Vertical margin of the appbar.
+/// @group appbar
+$kendo-appbar-margin-y: 0 !default;
+/// Horizontal margin of the appbar.
+/// @group appbar
+$kendo-appbar-margin-x: 0 !default;
+/// Vertical padding of the appbar.
+/// @group appbar
+$kendo-appbar-padding-y: map-get($kendo-spacing, 2) !default;
+/// Horizontal padding of the appbar.
+/// @group appbar
+$kendo-appbar-padding-x: map-get($kendo-spacing, 2) !default;
+/// Width of the border around the appbar.
+/// @group appbar
+$kendo-appbar-border-width: 0px !default;
+
+/// Z-index of the appbar.
+/// @group appbar
+$kendo-appbar-zindex: 1000 !default;
+
+/// Font size of the appbar.
+/// @group appbar
+$kendo-appbar-font-size: $kendo-font-size !default;
```

use the --kendo-font-size var:

```suggestion
$kendo-appbar-font-size: var( --kendo-font-size, inherit )!default;
```